collector/src/compile/execute/mod.rs

```diff
@@ -329,9 +329,16 @@ impl<'a> CargoProcess<'a> {
             let mut cmd = self.base_command(self.cwd, cargo_subcommand);
             cmd.arg("-p").arg(self.get_pkgid(self.cwd)?);
             match self.profile {
-                Profile::Check | Profile::Clippy => {
+                Profile::Check => {
                     cmd.arg("--profile").arg("check");
                 }
+                Profile::Clippy => {
+                    cmd.arg("--profile").arg("check");
+                    // Make sure that we run all lints, or else would 
+                    // be pointless for allow-by-default lint benchmarks
+                    // and would cause errors with deny-by-default lints.
+                    cmd.env("RUSTFLAGS", "-Wclippy::all");
```

Comment:
This overrides externally set rustflags, but it shouldn't be a big deal for the Clippy benchmarks. Just noting it here if anyone runs into it.

Reply:
In this new patch we even append instead of overwriting! :)